apps/admin/backend/src/app.test.ts

```diff
@@ -186,18 +187,38 @@ test('getSystemSettings happy path', async () => {
   expect(systemSettingsResult).toEqual(JSON.parse(systemSettings.asText()));
 });
 
-test('getSystemSettings returns default system settings when no system settings are found', async () => {
-  const { apiClient, auth } = buildTestEnvironment();
-
-  const { electionDefinition } = electionMinimalExhaustiveSampleFixtures;
-  await configureMachine(apiClient, auth, electionDefinition);
-
-  mockSystemAdministratorAuth(auth);
+test('getSystemSettings returns default system settings when there is no current election', async () => {
+  const { apiClient } = buildTestEnvironment();
 
   const systemSettingsResult = await apiClient.getSystemSettings();
   expect(systemSettingsResult).toEqual(DEFAULT_SYSTEM_SETTINGS);
 });
 
+test('getSystemSettings failure path', async () => {
+  const { apiClient, workspace, logger } = buildTestEnvironment();
+  const { store } = workspace;
+  const {
+    electionDefinition: { electionData },
+  } = electionMinimalExhaustiveSampleFixtures;
+
+  // configure with malformed system settings data, which is the only way for
+  // this path to fail
+  const electionId = store.addElection({
+    electionData,
+    systemSettingsData: '{ bad json }',
+  });
+  store.setCurrentElectionId(electionId);
```

Comment:
Can this fail if we give a nonexistent election ID?

Reply:
It would fail if we gave it a nonexistent election ID, but that would break most of the other endpoints too. I like to make the assumption that the current election ID always exists, otherwise we end up coding for a bunch of unrealistic failure cases.